src/common/crypto/ed25519_algebra/ed25519_algebra.c

```diff
@@ -3,7 +3,21 @@
 #include "crypto/keccak1600/keccak1600.h"
 #include "curve25519.c"
 
+#if HAVE_BYTESWAP_H
 #include <byteswap.h>
+#else
```

Comment:
is this for macOS? There is multiple changes related to `byteswap.h` in the PR and there is a replacement header that I use for macOS:

$ cat byteswap.h 
```
#ifndef __BYTESWAP_H__
#define __BYTESWAP_H__

#ifdef __cplusplus
extern "C" {
#endif //__cplusplus

// Mac OS X / Darwin features
#include <libkern/OSByteOrder.h>
#define __bswap_16(x) OSSwapInt16(x)
#define __bswap_32(x) OSSwapInt32(x)
#define __bswap_64(x) OSSwapInt64(x)

#ifdef __cplusplus
}
#endif //__cplusplus
#endif // __BYTESWAP_H__
```